modules/instruments/.gitignore

```diff
@@ -0,0 +1 @@
+js/CandidateInstrumentList.js
```

Comment:
I don't know about the imaging_browser one, but this one can just be `js/*`. It's a newish module and doesn't have any non-compiled js grandfathered in..

Reply:
hahaha every PR we debate the same thing 😂 

I don't like jokers 🃏